newrelic-agent/src/main/java/com/newrelic/agent/profile/v2/Murmur3StringMap.java

```diff
@@ -42,8 +41,9 @@ public Object addString(String string) {
             key = string;
         } else {
             HashCode hash = hashFunction.newHasher().putString(string, charSet).hash();
-            byte[] asBytes = hash.asBytes();
-            key = new String(base64.encode(asBytes, 0, 8), StandardCharsets.US_ASCII);
+            final byte[] eightBytes = new byte[8];
```

Comment:
local allocation for thread-safety. Good news, though, as `asBytes()` calls `clone`: https://github.com/google/guava/blob/master/guava/src/com/google/common/hash/HashCode.java#L265, so there should be no change in allocations.